example/Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.5.1'
+gem 'rails', '~> 4.2'
 gem 'sqlite3'
 
 gem 'dotenv-rails'
-gem 'shopify_app', '~> 7.0.0'
+gem 'shopify_app', '~> 7.2.0'
```

Comment:
:+1:  for removing the ShopifyApp version.
But I'm on the fence regarding the addition of Ruby version. When a Ruby version is specified, *Heroku* for example, uses only that version. Also should a user on a newer version, download the example app and start the local server, as is, there will be conflicts raised along the lines:
```
 You are using Ruby 2.3.1 but the Gemfile specified 2.2.2 
```

( I've nt tested this myself though )